### ISSUE TITLE
FE: Homepage template cleanup, removal of old markup

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#13955]
### User story

**AS A** PW Engineer
**I WANT** old homepage markup and conditional logic around launch flag removed
**SO THAT** we reduce tech debt and associated overhead.

### Engineering notes / background

This ticket paves the way for #11921. 

See that ticket for details and context (e.g., more explanation of "Homepage-related fields in Ben Hub landing page content type").

#### Benefit Hub Landing page fields
https://prod.cms.va.gov/node/3071/edit
This form includes 2 fields that were previously used to populate the "Explore VA benefits and health care" blurbs on the homepage, for each Benefit. This content is now managed by a Drupal menu. So _if_ these 2 fields are no longer used in any FE templates, they could be removed from Drupal.

Fields: 
1. Hub label - `field-home-page-hub-label-0`
2. Hub teaser text - `field-teaser-text-0`

![Screenshot 2023-06-02 at 11 03 33 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/85581471/e3a28a3e-22c0-4519-aa42-34a345081177)

## Acceptance criteria

- [x] Remove New Homepage invitation's modal markup from "announcement framework," but leave framework intact for future
- [x] Removed conditional template code, including markup for Old Homepage which we left in case of rollback
- [x] Sleuth whether Homepage-related fields in Ben Hub landing page content type are used in any other FE templates (besides OldHP), as they could cause content-build to fail. e.g. https://prod.cms.va.gov/node/3071/edit
  - [x] Update ACs on BE tkt 11921 if there is an issue found 
- [x] Namespace cleanup – updated variable names, file names, etc to align around "2023-homepage" or similar, so that future homepage iterations will be easy to disambiguate
  - [x] Raised new scheme with team to ensure it's clear

**Comment by jilladams** [comment]
Talked to Randi about how we figure out if the Label / Teaser fields from Benefit Hub Landing Pages are used multiple places. Summarizing what we figured out here, and will need @dsasser to confirm if we're right or not.  

**tl;dr:** VA Benefit Hub teaser text / label appear on many other places throughout the site.

**What we want to find:** 
how are the Homepage Hub Label and Teaser named when they come from the CMS through GraphQL into the front end. That naming will be consistent, and will help us know if the fields are used in multiple places / across multiple content types or templates. 

https://github.com/department-of-veterans-affairs/content-build/blob/b280c65f129eba681cf15a46a598bdca490bfab5/src/site/stages/build/drupal/home.js#L26 - is state of homepage pre-processing for the V1 homepage.  (We are now on V2.)
* data appears to come from `homePageHubListMenuQuery`

https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/stages/build/drupal/graphql/homePage.graphql.js
* the fields we care about are described in here as:
    * `fieldHomePageHubLabel` 
    * `fieldTeaserText`

When we search content-build for those: 
`fieldHomePageHubLabel`  = 14 hits: https://github.com/search?q=repo%3Adepartment-of-veterans-affairs%2Fcontent-build%20fieldHomePageHubLabel&type=code
`fieldTeaserText` = 13 hits: https://github.com/search?q=repo%3Adepartment-of-veterans-affairs%2Fcontent-build%20fieldTeaserText&type=code


https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/includes/benefit-hubs-links.drupal.liquid
* A template for "Related Benefit Hubs" 
* It looks like the `fieldRelatedBenefitsHubs` content uses both of these fields and their data across multiple templates on the frontend, including: Resources & Support, FAQs, Checklists, Media, Forms, Contact info paragraphs, Step by Steps. 

e.g. 
https://prod.cms.va.gov/node/47278/edit - Editor selects VA Benefit relationships on the node
![Screenshot 2023-06-02 at 3 23 47 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/85581471/8adcee69-2f7b-408b-b670-13f7ed776784)

https://www.va.gov/resources/the-pact-act-and-your-va-benefits/ - those appear on the FE under VA Benefits heading, at the bottom of the main content section, right above "Need more help".
![Screenshot 2023-06-02 at 3 21 17 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/85581471/d265ba84-c069-4def-ba68-c2c6b787dbee)


So, unfortunate: we now have 2 sources of truth for this content, Hub menu, and the Benefit Hub Landing Page fields. Womp. @wesrowe FYI. We should probably file a ticket for figuring out what to do about single source of truthing that. :/

**Comment by wesrowe** [comment]
I marked the ACs on the BE ticket to reflect that removal is not an option. Since these fields are being used elsewhere, it seems like what we have here is design debt – the fields are not being used how they're labeled in the CMS UI. Solution might be to rename the field set? 

Proposed change:

**Field set heading**
Old: Home page listing
New: Brief listing? Brief summary?

**Help text below heading**
Old: Add information for the Benefit Hub's listing on the VA.gov home page
New: Add information for this hub to be displayed when it's included in a list.

@DanielleThierryUSDSVA, do you have an opinion about what these blurbs could be called without referencing the Homepage? (See image in ticket description for context.) The comment above this one has examples of where else these short names and descriptions are used.

cc @thejordanwood

**Comment by randimays** [comment]
Will carry over to tomorrow as the deploy will not be happening until this afternoon (for maximum time validating in staging).

**Comment by randimays** [comment]
Jill and I verified in production yesterday. We're good to close!
